amid/ct_ich.py

```diff
@@ -92,13 +92,15 @@ def _patient_metadata(_root: Silent):
     def _diagnosis_metadata(_root: Silent):
         return pd.read_csv(Path(_root) / 'hemorrhage_diagnosis_raw_ct.csv')
 
-    def age(i, _patient_metadata):
-        num_id = int(i.split('_')[-1])
-        _patient_metadata['Age\n(years)'].loc[num_id]
+    def _row(i, _patient_metadata):
+        patient_id = int(i.split('_')[-1])
+        return _patient_metadata.loc[patient_id]
 
-    def gender(i, _patient_metadata):
```

Comment:
should probably keep the field for a while. just not to break backwards compatibility. You can move it to the `normalized` part